src/ui/atoms/box.js

```diff
@@ -0,0 +1,58 @@
+import styled, { css } from "styled-components"
+
+export const Box = styled.div`
+  margin: 0.5rem;
+  display: flex;
+  flex-flow: column;
+  flex-shrink: 0;
+  ${({ theme }) => theme.embed.card}
+  border: 1px solid rgba(0, 0, 0, 0.2);
+  border-radius: 4px;
+  padding: 2rem;
+  box-sizing: border-box;
+
+  &: hover {
+    display: flex;
+    flex-flow: column;
+    flex-shrink: 0;
+    border-radius: 4px;
+    padding: 2rem;
+    box-sizing: border-box;
+    box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.2);
+    //transform: scale(1.001);
+  }
+  ${({ sticky }) =>
```

Comment:
You really need `sticky` here?

Reply:
I would like to replace Card-atom with Box-atom later. Keep name "card" just for feature-components and Box as styled component. So later I am going to remove sticky from Card with whole Card-atom.